build broken with latest sdk (4.22.0)

```
compile 'com.facebook.android:audience-network-sdk:4.+'
```

The library dependency refers to all 4.* version
it's broken at 4.22.0 (fine with 4.20.0)

```
/Users/elvis/work/XXX/node_modules/react-native-fbads/src/android/src/main/java/io/callstack/react/fbads/BannerView.java:21: error: BannerView is not abst
ract and does not override abstract method onLoggingImpression(Ad) in AdListener
public class BannerView extends ReactViewGroup implements AdListener, LifecycleEventListener {
       ^
/Users/elvis/work/XXX/node_modules/react-native-fbads/src/android/src/main/java/io/callstack/react/fbads/InterstitialAdManager.java:13: error: Interstitia
lAdManager is not abstract and does not override abstract method onLoggingImpression(Ad) in AdListener
public class InterstitialAdManager extends ReactContextBaseJavaModule implements InterstitialAdListener, LifecycleEventListener {
       ^
Note: /Users/elvis/work/cybertool/node_modules/react-native-fbads/src/android/src/main/java/io/callstack/react/fbads/BannerViewManager.java uses unchecked or un
safe operations.
```

--- a/src/android/build.gradle
+++ b/src/android/build.gradle
@@ -36,5 +36,5 @@ repositories {
 dependencies {
     compile 'com.facebook.react:react-native:+'
     compile 'com.android.support:appcompat-v7:23.1.0'
-    compile 'com.facebook.android:audience-network-sdk:4.+'
+    compile 'com.facebook.android:audience-network-sdk:4.20.+'
 }

